model/page.go

```diff
@@ -19,7 +19,6 @@ type Page struct {
 	ShowFeedbackForm                 bool           `json:"show_feedback_form"`
 	ReleaseDate                      string         `json:"release_date"`
 	BetaBannerEnabled                bool           `json:"beta_banner_enabled"`
-	EnableCookiesControl             bool           `json:"enable_cookies_control"`
```

Comment:
Is this the only place this value is used?

Reply:
It's used in the renderer too. I've got a PR for removing its usage there as well which has been checked and approved